packages/shared/src/components/squads/SquadEmptyScreen.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { ReactElement, useContext } from 'react';
+import SettingsContext, { ThemeMode } from '../../contexts/SettingsContext';
+import useMedia from '../../hooks/useMedia';
+import { cloudinary } from '../../lib/image';
+import { tablet } from '../../styles/media';
+import { Image } from '../image/Image';
+import { FlexCentered } from '../utilities';
+
+function SquadEmptyScreen(): ReactElement {
+  const settings = useContext(SettingsContext);
+  const isMobile = !useMedia([tablet.replace('@media ', '')], [true], false);
+
+  return (
+    <FlexCentered className="flex-col tablet:self-center py-14 w-full laptop:h-full text-center laptop:min-h-[calc(100vh-442px)]">
+      <Image
+        className="mb-8"
+        width={isMobile ? 180 : 314}
+        height={isMobile ? 91 : 152}
+        src={
+          settings.themeMode === ThemeMode.Light
```

Comment:
@rebelchris is this fine for dark/light mode handling? From what I can see we kinda fallback to the dark if no preference?

Reply:
Hmm I see we actually never switch images so far.
Think this is fine from my point of view, @sshanzel you see any concerns?